packages/react-components/react-storybook-addon-export-to-sandbox/package.json

```diff
@@ -24,7 +24,11 @@
   "peerDependencies": {
     "@storybook/manager-api": "^7.6.20",
     "@storybook/preview-api": "^7.6.20",
-    "@storybook/react": "^7.6.20"
+    "@storybook/react": "^7.6.20",
+    "@types/react-dom": ">=16.9.0 <19.0.0",
```

Comment:
this is antipattern present in all v9 UI related controls, types should be only optional peers